be/src/olap/rowset/segment_v2/page_builder.h

```diff
@@ -91,5 +94,15 @@ class PageBuilder {
     DISALLOW_COPY_AND_ASSIGN(PageBuilder);
 };
 
+template <typename Derived>
+class PageBuilderHelper : public PageBuilder {
+public:
+    template <typename... Args>
+    static Status create(PageBuilder** builder, Args&&... args) {
+        *builder = new Derived(std::forward<Args>(args)...);
```

Comment:
use a unique ptr
if init failed, then it is auto released.
if status is error, user may not release the builder manually.